client/web/antrea-ui/src/utils/fakeData.ts

```diff
@@ -0,0 +1,113 @@
+import { AgentInfo } from '../api/info';
+
+export function generateFakeAgents(n: number): AgentInfo[] {
+    // Create node indices in sequential order
+    const nodeIndices = Array.from({ length: n }, (_, i) => i);
+    
+    // Shuffle the indices to randomize the order of agent names
+    const shuffledIndices = [...nodeIndices].sort(() => Math.random() - 0.5);
+    
+    // Map of all possible node numbers to ensure we generate all nodes (1 to n)
+    // but display them in random order
+    const generateAgentData = (actualIndex: number, shuffledIndex: number) => {
+        // Helper function to generate structured agent names
+        const generateAgentName = () => {
+            // Use more inclusive terminology (replaced 'master' with 'control-plane')
+            const nodeTypes = ['control-plane', 'worker'];
+            // First node is control-plane, others are workers
+            const nodeType = actualIndex === 0 ? nodeTypes[0] : nodeTypes[1];
+            // For worker nodes, use sequential numbering based on actual index
+            const nodeNumber = actualIndex === 0 ? 1 : actualIndex;
+            return `k8s-node-${nodeType}-${nodeNumber}`;
+        };
+
+        // Helper function to generate specific versions (only two possible versions)
+        const generateVersion = () => {
+            // Use only two possible versions to simulate a cluster during upgrade
+            return actualIndex % 3 === 0 ? 'v2.3.0' : 'v2.2.3';
+        };
+
+        // Helper function to generate recent timestamps within last 5 minutes
+        const generateTimestamp = () => {
+            const now = new Date();
+            const fiveMinutesAgo = new Date(now.getTime() - (5 * 60 * 1000));
+            const randomTime = new Date(
+                fiveMinutesAgo.getTime() + Math.random() * (now.getTime() - fiveMinutesAgo.getTime())
+            );
+            return randomTime.toISOString();
+        };
+
+        // Helper function to generate consecutive subnets
+        const generateSubnets = () => {
+            const count = Math.min(actualIndex % 2 + 1, 2); // 1-2 subnets, more predictable
+            const subnets = [];
+            
+            // Generate sequential IPv4 subnets
+            if (count > 0) {
+                // Base subnet with sequential third octet
+                subnets.push(`10.10.${actualIndex % 255}.0/24`);
```

Comment:
If you have too many fake agents, you will have duplicate subnets. Should there be an assertion to limit the number of agents?